src/styles/main.scss

```
@@ -1,3 +1,31 @@
 body {
   margin: 0;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+$blockSize: 300px;
+$blockColors: 1 #f00, 2 #c00, 3 #900, 4 #600, 5 #300, 6 #000;
```

```suggestion
$blockColors:
1 #f00,
2 #c00,
3 #900,
4 #600,
5 #300,
6 #000;
```
Might better you to put it in _variables.scss